asyncwrapper/src/main/kotlin/org/n/riesgos/asyncwrapper/dummy/AbstractWrapper.kt

```diff
@@ -346,7 +346,7 @@ abstract class AbstractWrapper(val publisher : PulsarPublisher, val wpsConfigura
         val wpsProcess = retry<WPSProcess>(wpsConfiguration.retryConfiguration.maxRetries, wpsConfiguration.retryConfiguration.backoffMillis, { ex -> ex is WPSClientException }) {
             LOGGER.info("retrieve getCapabilities document from  ${wpsConfiguration.wpsURL} (retries: $it)")
             val wpsClientService = WPSClientService(wpsConfiguration)
-            val wpsProcess = WPSProcess(wpsClientService.establishWPSConnection(), getWpsUrl(), getWpsIdentifier(), "2.0.0", getWpsDialect(), getRequestedOutputs(),wpsConfiguration.retryConfiguration)
+            val wpsProcess = WPSProcess(wpsClientService.establishWPSConnection(), getWpsUrl(), getWpsIdentifier(), wpsConfiguration.wpsVersion, getWpsDialect(), getRequestedOutputs(),wpsConfiguration.retryConfiguration)
```

Comment:
@nbrinckm 
Is there any reason to use fixed version 2.0.0 to run wps processes?

Reply:
I think it was just because all of our gfz services were wps 2.0.0. I'm also not 100% sure how the mapper logic (from wps input-output classes  to wrapper input-output classes) is specific for 2.0.0.